Android/Diwakar Prajapati/Newsopedia/app/src/main/java/com/example/newsopedia/MainActivity.java

```diff
@@ -0,0 +1,69 @@
+package com.example.newsopedia;
+
+import android.os.Bundle;
+import android.view.View;
+import android.view.Menu;
+
+import com.example.newsopedia.Model.Articles;

```

Comment:
Please clean the code and remove unused imports, methods, variables, print& debug statements.

Reply:
Done